tests/parser/parser.cpp

```diff
@@ -1471,23 +1470,26 @@ TEST(Parser, parseIdsOnEverything)
         "  <units name=\"units3\" id=\"u3id\"/>\n"
         "  <component name=\"component2\" id=\"c2id\">\n"
         "    <variable name=\"variable1\" units=\"blob\" id=\"v1id\"/>\n"
-        "    <reset variable=\"variable1\" order=\"1\" id=\"r1id\">\n"
-        "      <when order=\"5\" id=\"w1id\">\n"
+        "    <variable name=\"variable2\" units=\"blob\" id=\"v2id\"/>\n"
+        "    <reset variable=\"variable1\" test_variable=\"variable2\" order=\"1\" id=\"r1id\">\n"
+        "      <test_value>\n"
```

Comment:
@nickerso I haven't seen anything either way, other than the contents of test_value and reset_value was to be mathml.  Is there a new version of the spec around resets yet? I've just been going from the use case document?  I'd imagine that the test_value needs to be an expression though? and that there could be cases where you'd want to reset to something related to another variable, so that the reset_value needs to be an expression too?  @MichaelClerx what are your thoughts?